go/processor/floria/processor.go

```diff
@@ -70,31 +73,74 @@ func (p *processor) Run(
 		// Create new contract
 	} else {
 		// Call existing contract
-		result, err = call(p.interpreter, transaction, context)
+		result, err = call(p.interpreter, transaction, context, gas)
 		if err != nil {
-			return errorReceipt, nil
+			return tosca.Receipt{GasUsed: transaction.GasLimit}, nil
```

Comment:
The potential `err` produced by the `call` is ignored. Should this be like this?
```suggestion
			return tosca.Receipt{GasUsed: transaction.GasLimit}, err
```

Reply:
Question: 
If we return `value, err` and the caller of this function does the usual:
 ```go 
 v, err :=call()
 if err {
       // pontentially break flow
}
 ```

Aren't we dropping the value in that case? 
wouldn't be nil, err more solid? 

The way I see it, there are two options: 

- processor takes the decision, this error is handled as `transaction.GasLimit`, log error is needed, after which it can be dropped
- Processor honours the error, returning `nil, err`, and error handling is applied in the caller side. 

The proposed change will lead to code ignoring the `transaction.GasLimit` resolution and end up having a new resolution for the error applied.